voluncheer/opportunityboard/views/opportunityboard.py

```diff
@@ -91,8 +92,13 @@ def deregister_volunteer(request, opportunity_id):
 
 def organization_view(request, userid):
```

Comment:
I don't understand why we are getting an organziaton, user and volunteer here. Can we just get the User and check if it's a volunteer or organization type?

Reply:
<img width="808" alt="image" src="https://user-images.githubusercontent.com/43869961/232245116-e8331e25-e62a-4382-ae82-55511d1fe981.png">
Referring to this, we need to make it clear that it is the volunteer user visiting organization for future uses. Now it is kinda messed up for both view and test.

Reply:
Ah, I see. I fixed this in my PR by adding a `curr_user` kwarg to the context. The template can then check if `curr_user is user` and include the correct email.

Reply:
That even better! Can you help me adapt this part to your model after merging then?

Reply:
Will do.